geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/SimpleFeatureOrdering.scala

```diff
@@ -71,6 +124,11 @@ object SimpleFeatureOrdering {
       nullCompare(property.evaluate(x).asInstanceOf[Comparable[Any]], property.evaluate(y))
   }
 
+  private class UserDataOrdering(key: String) extends Ordering[SimpleFeature] {
```

Comment:
Is this used?

Reply:
no - I was initially playing around with stuffing the sort field into the user data so that we wouldn't have to return it, so I figured i'd leave it here in case we want to pursue that later. I can remove it if you want.